openapi-generator/templates/go/README.mustache

```diff
@@ -41,6 +41,13 @@ func main() {
 }
 ```
 
+# Using US Datacenter
+In order to use the US Datacenter for your requests, set the `BasePath` of the configuration to `"https://us.api.phrase.com/v2"` like so:
+
+```
+cfg.SetBasePath("https://us.api.phrase.com/v2")
```

Comment:
I think it should be `https://app.us.phrase.com/api/v2` if I'm not mistaken

Reply:
Yes, `app.us`.